community-roadmap/frontend/README.md

```diff
@@ -0,0 +1,32 @@
+# Community Roadmap for Zendesk Help Center
+
+This folder contains the frontend for the Zendesk community roadmap project.
+The frontend is built with npm and React.
```

Comment:
Let's use a different heading and assume the reader already read the top-level README.

I think we need to provide more context about what the front-end service is doing and how it interacts with the backend.